tests/exporters/onnx/test_onnx_export.py

```diff
@@ -314,6 +311,30 @@ def _onnx_export(
 
                 gc.collect()
 
+    def _onnx_export_sd(self, model_type: str, model_name: str, device="cpu"):
```

Comment:
I think the only benefit of this `test_onnx_export.py` is to test on various shapes the exported model.

Couldn't we just remove those tests for SD? Does it bring anything more than the tests you put in `test_exporters_onnx_cli`?

Reply:
In `test_onnx_export.py` we verify whether each model's subcomponent is correctly exported by comparing the output of the original subcomponent with the exported one : https://github.com/huggingface/optimum/blob/v1.9.1/tests/exporters/onnx/test_onnx_export.py#L411